packages/react/src/components/form/store-recovery-codes/store-recovery-codes.css.ts

```diff
@@ -0,0 +1,12 @@
+import { globalStyle, style } from "@vanilla-extract/css";
+
+export const recoveryCodes = style({
+  marginTop: 24,
+});
+
+globalStyle(`${recoveryCodes} > div`, {
```

Comment:
This can break, we can work around it this way:
- expose a prop on the `ReadOnlyField` like `fieldClassName`
- then apply it to the child element, regardless of the element type

What do you think?